wp-content/themes/csisjti/inc/template-functions.php

```diff
@@ -415,3 +415,14 @@ function csisjti_remove_easy_footnotes($content) {
     }
     add_filter('the_content', 'csisjti_remove_easy_footnotes', 20);
 }
+
+function the_field_without_wpautop( $field_name ) {
```

Comment:
Can you add a comment about what this function does? Take a look at some of the other functions for an example of what to include. It'll help future-us remember. :)